Task3/js/app.js

```diff
@@ -0,0 +1,42 @@
+const container = document.querySelector('.zombies');
```

Comment:
these whole variables and functions are global (declared in window)

Reply:
I suppose, it have to be the global, haven't it?

Reply:
nope
I mean these variables are accessible in any script, but it shouldn't be like this
use DOMContentLoaded event to fix it

Reply:
If I use  DOMContentLoaded event in app.js I can't use variable "Counter" and function createZombie() from Zombie.js. Do I need to call variable "Counter" and function createZombie()  only in app.js?

Reply:
yep
1) "counter" shouldn't be a global var
2) "createZombie" shouldn't be a global func